plugins/opsgenie/plugin.spec.yaml

```diff
@@ -0,0 +1,277 @@
+    title: Get Alert
+    description: Retrieve alert from OpsGenie
+    input:
+      identifier:
+        title: Alert ID, Tiny ID or Alias
+        description: Identifier of the alert
+        type: string
+        required: true
+        example: "8418d193-2dab-4490-b331-8c02cdd196b7"
+      identifierType:
+        title: Identifier Type
+        description: Type of the identifier that is provided as an in-line parameter. Possible values are ID, tiny ID and alias. Default value is ID
+        type: string
+        enum:
+          - ""
+          - "ID"
+          - "tiny"
+          - "alias"
+        default: "ID"
+        required: false
+        example: "ID"
+    output:
+      data:
+        title: Data
+        description: Data that contains JSON response
+        type: object
+        required: false
+      requestId:
+        title: Request ID
+        description: ID of an request
+        type: string
+        required: false
+  close_alert:
+    title: Close Alert
+    description: Close an existing alert from OpsGenie
+    input:
+      identifier:
+        title: Alert ID, Tiny ID or Alias
+        description: Identifier of the alert
+        type: string
+        required: true
+        example: "8418d193-2dab-4490-b331-8c02cdd196b7"
+      identifierType:
+        title: Identifier Type
+        description: Type of the identifier that is provided as an in-line parameter. Possible values are ID, tiny ID and alias. Default value is ID
+        type: string
+        enum:
+          - ""
+          - "ID"
+          - "tiny"
+          - "alias"
+        default: "ID"
+        required: false
+        example: "ID"
+      user:
+        title: User
+        description: Display name of the request owner
+        type: string
+        required: false
+        example: "Monitoring Script"
+      source:
+        title: Source
+        description: Display name of the request source
+        type: string
+        required: false
+        example: "AWS Lambda"
+      note:
+        title: Note
+        description: Additional alert note to add
+        type: string
+        required: false
+        example: "Action executed via Alert API"
+    output:
+      result:
+        title: Result
+        description: Result message from API
+        type: string
+        required: false
+      took:
+        title: Took
+        description: Time took to execute API
+        type: float
+        required: false
+      requestId:
+        title: Request ID
+        description: ID of a executed API request
+        type: string
+        required: false
+  get_on_calls:
+    title: Get On-Calls
+    description: Get on-call request is used to retrieve current on-call participants of a specific schedule
+    input:
+      scheduleIdentifier:
+        title: Schedule Identifier
+        description: Identifier of the schedule
+        type: string
+        required: true
+        example: "ScheduleName"
+      scheduleIdentifierType:
```

Comment:
Same as above with identifier types - this should be auto-determined via code if possible instead of making this an extra input for the user to fill out.